webxrlayers-1.bs

```diff
@@ -1446,6 +1446,10 @@ If the [=XR Compositor=] is driving a monoscopic display and the rendered {{XRCo
 If the {{XRCompositionLayer}} was created through {{XRMediaBinding}} and the {{XRMediaLayerInit/"invertStereo"}} flag was set to
 <code>true</code>, the bottom portion of the {{XRLayer}} is shown while the top portion is ignored.
 
+If the [=XR Compositor=] is driving a additional display in addition to a monoscopic or stereoscopic one, it is up to the
+user agent how a {{XRCompositionLayer}} with an {{XRLayerLayout}} of {{XRLayerLayout/"stereo-left-right"}} or
+{{XRLayerLayout/"stereo-top-bottom"}} is shown to that additional display.
```

Comment:
Do we need language about `mono` too?

Reply:
Isn't that already covered by the 2 previous paragraphs?

Reply:
is it, I don't see any text about how to render mono layers. I mean it's kind of obvious, so maybe it doesn't need spelled out?

Reply:
Yes, I don't think it's needed.